docs/whats-new.md

```diff
@@ -57,18 +57,20 @@ A new method `CompositeLayer.getBaseLayerProps()` simplifies forwarding base lay
 
 ## PathLayer & GeoJsonLayer: Dashed Line Support
 
-Added new props (`getDashArray` and `justified`) enabling you render paths as dashed lines. Naturally these props are also accessible in composite layers built on top of the `PathLayer`, such as the `GeoJsonLayer`.
+Added new props (`getDashArray` and `dashJustified`) enabling you render paths as dashed lines. Naturally these props are also accessible in composite layers built on top of the `PathLayer`, such as the `GeoJsonLayer`.
 
 
 ## HexagonLayer / GridLayer: Elevation by Value Support
 
 Add `getElevationValue` to `HexagonLayer` and `GridLayer` to enable elevation aggregation by value. This allow both color and elevation to be calculated based on customized aggregation function.
 
+## Query methods renamed
```

Comment:
I'd call the section Picking methods renames 

This could be grouped with layer filtering as both relate to props on deck.gl's React interface.

Reply:
`layer Filtering` is a prop but these are methods, will update the title.

Reply:
The weird thing is that they both relate to the same component but are separated by sections about other components, i.e. layers. I was just suggesting reordering the sections.